### PR TITLE
EI-1257 Do not expect seeds given in auth example

### PR DESCRIPTION
**Pull request by bkr-iotic**

### CODE REVIEW

**Comment by bkr-iotic** [comment]
Actions will not succeed until the secrets are changed, and we should not change the secrets until we agree here that this is a good way to demonstrate authentication. Running `make run-examples` should work locally with an appropriate `.env` file, which I can provide.

File: examples/identity_auth.py [hunk 1]
```diff
@@ -41,33 +38,63 @@ def __init__(
...
         return self.grpc_token
 
+    def refresh_token(self, ttl: typing.Optional[int] = None):
+        self.api.create_agent_auth_token(self.agent, self.user_did, ttl or self.token_ttl)
+
     def generate_twin_did(self, twin_name) -> str:
         # Reuse the agent secret seed for simplicity and ease of twin maintenance.
-        seed = self.agent_registered_id.key_pair_secrets.seed
+        seed = self.agent.key_pair_secrets.seed
         # To keep your twins secure, you should ensure that the twin key name is secret and not easy to guess,
```
Comment (captnz) [inline examples/identity_auth.py:73]: Perhaps change `twin key name` to `twin_name` in comment?

**Review by captnz — APPROVED** [review]
lgtm wfm
Update unreleased section of changelog with changes?
Also need to pass by @Elleppi before releasing?

**Comment by Elleppi** [comment]
As far as I'm concerned, no one is currently using this library as yet so feel free to make the release (let me know if this is not going to happen soon)

File: examples/identity_auth.py (hunk 1 above)
Comment (bkr-iotic) [reply examples/identity_auth.py:73]: I don't think that needs to be kept secret (usually I just use the label) so I've removed the comment entirely

**Pull request merged**